DSTV.Net/Implementations/BodyReader.cs

```diff
@@ -55,6 +57,7 @@ public static async Task<IEnumerable<DstvElement>> GetElementsAsync(ReaderContex
             foreach (var bendNote in bendBlocks.SelectMany(bendList => bendList))
                 try
                 {
+                    if (bendNote.Equals(ContourType.KA.ToString(), StringComparison.Ordinal)) continue;
```

Comment:
I feel like there is a lot of repetition going on here...should we make a generic function for this?

Reply:
I've simplified the code